<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>702</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/images/icon.png</normaloff>:/images/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="10">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>166</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="5" colspan="6">
     <widget class="QGraphicsView" name="graphicsView_R">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="BN_1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>&gt;&lt;&gt;</string>
      </property>
      <property name="icon">
       <iconset resource="names.qrc">
        <normaloff>:/resources/cut.png</normaloff>:/resources/cut.png</iconset>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <spacer name="horizontalSpacer_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>166</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="7">
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>167</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>167</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="BN_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>&gt;&gt;</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="5">
     <widget class="QGraphicsView" name="graphicsView_L">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <widget class="QPushButton" name="BN_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>&lt;&lt;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="8">
     <widget class="QPushButton" name="BN_4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>&lt;&gt;&lt;</string>
      </property>
      <property name="icon">
       <iconset resource="names.qrc">
        <normaloff>:/resources/open.png</normaloff>:/resources/open.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>999</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile_F">
    <property name="geometry">
     <rect>
      <x>496</x>
      <y>174</y>
      <width>175</width>
      <height>150</height>
     </rect>
    </property>
    <property name="title">
     <string>Model(M)</string>
    </property>
    <addaction name="action_Open_cfg"/>
    <addaction name="action_Open_weight"/>
    <addaction name="action_Open_class"/>
    <addaction name="action_Exit_Q"/>
   </widget>
   <widget class="QMenu" name="menuhelp">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>174</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>File(F)</string>
    </property>
    <addaction name="action_pic"/>
    <addaction name="action_video"/>
    <addaction name="actionVoc"/>
    <addaction name="actionCOCO"/>
    <addaction name="actionDOTA"/>
    <addaction name="actionCamera"/>
   </widget>
   <widget class="QMenu" name="menuLabel">
    <property name="title">
     <string>Label</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTrain">
    <property name="title">
     <string>Train</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTest">
    <property name="title">
     <string>Test</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEval">
    <property name="title">
     <string>Eval</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuOptim">
    <property name="title">
     <string>Optim</string>
    </property>
   </widget>
   <addaction name="menuhelp"/>
   <addaction name="menuLabel"/>
   <addaction name="menuTrain"/>
   <addaction name="menufile_F"/>
   <addaction name="menuTest"/>
   <addaction name="menuEval"/>
   <addaction name="menuOptim"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>173</red>
         <green>127</green>
         <blue>168</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>173</red>
         <green>127</green>
         <blue>168</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>190</red>
         <green>190</green>
         <blue>190</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
  </widget>
  <action name="action_Open_cfg">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Open_cfg</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_Open_weight">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Open_weight</string>
   </property>
  </action>
  <action name="action_Open_class">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Open_class</string>
   </property>
  </action>
  <action name="action_Exit_Q">
   <property name="text">
    <string>Exit(Q)</string>
   </property>
  </action>
  <action name="action_pic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Pic</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="action_video">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Video</string>
   </property>
  </action>
  <action name="actionVoc">
   <property name="text">
    <string>VOC</string>
   </property>
  </action>
  <action name="actionCOCO">
   <property name="text">
    <string>COCO</string>
   </property>
  </action>
  <action name="actionDOTA">
   <property name="text">
    <string>DOTA</string>
   </property>
  </action>
  <action name="actionCamera">
   <property name="text">
    <string>Camera</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="names.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_Exit_Q</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
